tests/test_bit_kernels.py

```diff
@@ -0,0 +1,74 @@
+"""
+Test suite for bit vector kernels.
+Author: Ryan-Rhys Griffiths 2021
+"""
+
+import pytest
+import torch
+
+from gprotorch.kernels.bit_vector_kernel import BitDistance
+
+
+@pytest.mark.parametrize("x1, x2", [
+    (torch.ones((2, 2)), torch.ones((2, 2))),
+    (2*torch.ones((2, 2)), 2*torch.ones((2, 2))),
+    (10*torch.ones((2, 2)), 10*torch.ones((2, 2))),
+    (10000*torch.ones((2, 2)), 10000*torch.ones((2, 2))),
+    (0.1*torch.ones((2, 2)), 0.1*torch.ones((2, 2)))
+])
+def test_tanimoto_similarity_with_equal_inputs(x1, x2):
+    """
+    Test the Tanimoto similarity metric between two equal input vectors.
+    """
+    dist_object = BitDistance()
+    tan_similarity = dist_object._tan_similarity(x1, x2, postprocess=False, x1_eq_x2=True)
+
+    assert (tan_similarity == torch.ones((2, 2))).all() == True
+
+
+def test_tanimoto_similarity_with_unequal_inputs():
+    """
+    Test the Tanimoto similarity metric between two unequal input vectors.
+    """
+    x1 = torch.ones((2, 2))
+    x2 = 2*torch.ones((2, 2))
+    dist_object = BitDistance()
+    tan_similarity = dist_object._tan_similarity(x1, x2, postprocess=False)
+
+    assert torch.allclose(tan_similarity, torch.tensor(0.6666666666666))
+
+
+# GPflow Tanimoto kernel implementation below
```

Comment:
This looks different to the Tanimoto implementation in bit_vector_kernel.py.  What is the rationale behind including this here?